sixMans/sixMans.py

```diff
@@ -868,6 +888,27 @@ async def monthly(self, ctx: Context, player: discord.Member = None, *, queue_na
         player = player if player else ctx.author
         await ctx.send(embed=self.embed_rank(player, sorted_players, queue_name, queue_max_size, "Monthly"))
 
+    @commands.guild_only()
+    @rank.command(aliases=["year", "yr"])
+    async def yearly(self, ctx: Context, player: discord.Member = None, *, queue_name: str = None):
```

Comment:
A little weird how qlb and rank swap grammar patterns between "yearly" and "year", "monthly" and "month", but its not a big deal. Maybe a small change we can make in the future.

Reply:
Yeah, I had never actually used the rank command before so wasn't certain why it followed a different flow. Didn't want to rock the boat too much and force that change in this PR, but happy to normalize these in a different ticket. (Are we, as a team, using tickets? Or just trello?)